### ISSUE TITLE
Help needed: Cassandra hosts not found, probably just a misconfiguration, Kong 0.10.0

### ISSUE THREAD

**Issue by Vermeille** [Mashape/kong#2239]
Hi!

We're migrating to Kong 0.10.0.

We're running Kong inside Docker.

In the configuration, cassandra_contact_points is correctly filled, dns_resolver is left blank, and so is dns_hostsfile.

However, starting kong generates those error messages (with s/real domain name/example.com/) 

> Error: [cassandra error] all hosts tried for query failed. node01.example.com:9042: host seems unhealthy, considering it down (host not found). node03.example.com:9042: host seems unhealthy, considering it down (host not found). node02.example.com:9042: host seems unhealthy, considering it down (host not found)

I can use netcat on those ports and talk to Cassandra. The hosts are reachable, and the domain names are resolved by the OS.

Any idea?

EDIT: Adding the raw IP in cassandra_contact_points does not work better, and the DNS is maybe out of cause. I have clearly no idea what's going on.

**Comment by Tieske** [comment]
does it fail and refuses to start, or does it start and then starts generating errors?

possible steps:

- Use FQDN names
- set a dns server, or update the one in resolv.conf

**Comment by Vermeille** [comment]
I just found the cause.

I specified the port in cassandra_contact_points like we used to do. Apparently you changed your policy about it (or the cassandra package did) since there is now cassandra_port to specify for all nodes. I migrated my configuration from 0.9.9 to 0.10.0 too quickly. Maybe there should be a warning about it in the Upgrade guide, or a check in kong when it parses the contact points that they do not include ':'. I can make  a PR for that.
